src/data/media/starwars.rs

```diff
@@ -52,6 +51,11 @@ pub fn vehicle() -> String {
 	let mut rng = rand::thread_rng();
 	VEHICLES[rng.gen_range(0..VEHICLES_LEN)].to_string()
 }
+
```

Comment:
The following functions should have #[wasm_bindgen] not just the first one.

Reply:
Sorry, I must of forgotten in the copy paste frenzy. I will fix it